src/main/java/org/dita/dost/platform/Integrator.java

```diff
@@ -321,7 +321,7 @@ private void integrate() throws Exception {
                 //configuration.put(name, ditaDir.getAbsolutePath());
                 configuration.put(name, ".");
             } else {
-                configuration.put(name, FileUtils.getRelativePath(new File(ditaDir, "dummy"), f.getPluginDir()).getPath());
+                configuration.put(name, FileUtils.getRelativePath(new File(ditaDir, "dummy"), f.getPluginDir()).getPath().replace("\\", "/"));
```

Comment:
If we convert this to always use `/`, then it might as well be a URI.

Reply:
There is getRelativeUnixPath that does the trick but it's deprecated.  Any specific reason?  
configuration.put(name, FileUtils.getRelativeUnixPath(aFile.toURI().toString(), f.getPluginDir().toURI().toString()));

Even if we were to create a method that takes URIs, it would still be calling the following underneath the covers.
private static String getRelativePath(final String basePath, final String refPath, final String sep) 

Suggestions?

Reply:
Ahh found this method in URLUtils

Reply:
The last commit is the good one.